### PR TITLE
Only append signature to file if no error

### LINKED ISSUE #18: Errors are dumped in the `signatures.txt` file
my `signatures.txt` file has the error messages dumped after having denied the signature:

image

### PR DESCRIPTION
**Pull request by Rubilmax**
Fixes #18

### CODE REVIEW

**Review by adhusson — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -23,6 +24,7 @@ exec\:%:
 	forge script script/ExecTransaction.s.sol --$* --broadcast
 
 clean:
+	mkdir -p data
```
Comment (MerlinEgalite) [inline Makefile:27]: but the `data` folder there right? It avoids issues if the user removes it unexpectedly right?

**Review by MerlinEgalite — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Rubilmax) [reply Makefile:27]: Indeed, and it was removed from the repository when adding the ts script so I ran into this issue.

**Review by julien-devatom — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -14,7 +14,8 @@ hash:
 	forge script script/HashData.s.sol
 
 sign\:%: hash
-	cast wallet sign --$* $$(cat data/hashData.txt) >> data/signatures.txt
+	OUTPUT=$$(cast wallet sign --$* $$(cat data/hashData.txt)) && echo "$$OUTPUT" >> data/signatures.txt
+	@echo "\033[0;32mTx signature successfully appended to data/signatures.txt"
```
[the remark this answers is not in the document]
Comment (Rubilmax) [reply Makefile:20]: Yes it is, but it won't get echoed out to the signatures file because of `&&`

[the remark this answers is not in the document]
Comment (julien-devatom) [reply Makefile:20]: You can simply use 2 different redirection for stderr & stdout:
```bash
cast wallet sign --$* $$(cat data/hashData.txt) 1>> data/signatures.txt 2>> /dev/null
```
Idk if it is working but it looks cleaner imo. Btw not a big deal, just a suggestion

**Review by MerlinEgalite — APPROVED** [review]

**Review by julien-devatom — APPROVED** [review]
Cleaner ahah 

🚀 🔥 LGTM 🔥 🚀

**Pull request merged**